python/src/runner.py

```diff
@@ -303,27 +332,66 @@ def main(globals):
     fileToRun = args.file
     if args.changeDir:
         os.chdir(os.path.dirname(fileToRun))
-    isInteractive = sys.flags.interactive
+    isInteractive = args.interactive
     version = readVersion()
     if isInteractive:
         prepareInteractive(reset=not args.noClear)
+    
     if not args.noInstall:
         installLib()
     if fileToRun is None:
         return
     if not args.checkRunnable and not args.quiet:
         printWelcomeString(fileToRun, version)
+
     libDefs = loadLib(onlyCheckRunnable=args.checkRunnable)
+
+    if not args.noUntypy:
+        mods = findModuleCandiates()
+        verbose("installing import hook on " + repr(mods))
+        untypy.just_install_hook(mods)
```

Comment:
@CodeSteak Why not starting at args.file for installing untypy's hook? Looking in the current directory does not work in all situations!

Reply:
fixed by using modulefinder (https://docs.python.org/3/library/modulefinder.html)

Reply:
Didn't thought of it. `args.file` seems to be the better solution.